app/src/main/java/mod/elfilibustero/sketch/lib/utils/CustomVariableUtil.java

```diff
@@ -0,0 +1,41 @@
+package mod.elfilibustero.sketch.lib.utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CustomVariableUtil {
+    public static final Pattern PATTERN_CUSTOM_VARIABLE = Pattern.compile("\\b((private|public|protected|static|final)\\s*)*([\\w$<>]+(?:\\s*,\\s*[\\w$<>]+)*(?:\\[\\])*)(?:\\s+([\\w$]+))(?:\\s*=\\s*(.*?))?\\b");
```

Comment:
for some reason `List<String> asd = new ArrayList<>()` isn't accepted

Reply:
and like in VariableTypeValidator, `?` isn't handled properly

Reply:
also looks like initializers also mustn't contain `()` or be literal Strings, otherwise it breaks